### ISSUE TITLE
[Bug] .profile is not read when logging in to "shell"

### ISSUE THREAD

**Issue by mchaker** [fairyglade/ly#676]
### Pre-requisites

- [X] I have looked for any other duplicate issues

### Ly version

1.1.0-dev.73+3706126

### Observed behavior

When logging in to "shell", `~/.profile` is not read. Without `ly`, both `~/.profile` and my shell-specific `~/.bashrc` are read in (env vars, PATH, etc.)

### Expected behavior

I expected `~/.profile` to be read in to the shell config, just like the system used to before installing `ly`.

### Steps to reproduce

1. Authenticate/log in to "shell"
2. Try to use env vars set in `~/.profile`, realize they are not set

### Relevant logs

```shell
I discovered this behavior by noticing that my PATH that is set in ~/.profile is not getting set when using ly. This leads me to believe that .profile is not getting read/executed by ly when logging in to the "shell" option.
```

**Comment by AnErrupTion** [comment]
That's strange, I thought Bash would automatically source these files at launch. Anyway, does launching manually the `/etc/ly/wsetup.sh` script right after logging in work? If it does, then a quick fix would be to make the shell session launch through this script (note that there isn't anything Wayland-specific to this script, it simply sources the appropriate files based on the current shell).

**Comment by mchaker** [comment]
Manually launching the `/etc/ly/wsetup.sh` script does not seem to load in the environment variables/PATH from `~/.profile`

**Comment by mchaker** [comment]
![image](https://github.com/user-attachments/assets/7460f293-dd2e-445b-8fa3-fa40ae83f05c)

**Comment by AnErrupTion** [comment]
@mchaker Can you check if the `SHELL` environment variable is set? Also, what's the output of `ls` in your home directory?

**Comment by mchaker** [comment]
![image](https://github.com/user-attachments/assets/7176f844-83aa-4c52-b0b8-e8675a5cb013)

**Comment by AnErrupTion** [comment]
@mchaker My bad, you need to make it run a new shell in order for the sourced files to be effective. So you can run it like so: `bash /etc/ly/wsetup.sh bash`. I tested it in a VM and it did work.

**Comment by mchaker** [comment]
Ah, `bash /etc/ly/wsetup.sh bash` does work after logging in!

**Comment by AnErrupTion** [comment]
@mchaker I've pushed the fix, can you confirm the issue is fixed with the latest commits?

**Comment by mchaker** [comment]
I can check in ~6 hours

**Comment by mchaker** [comment]
The issue is fixed as of this version: `1.1.0-dev.81+5e85618`

Thank you so much!
